tests/addon/test_unit.py

```diff
@@ -3055,62 +3059,80 @@ def test_send_daily_review_summaries_without_data(mocker):
     mock_anki_hub_client.send_daily_card_review_summaries.assert_not_called()
 
 
+def url_mh_integrations_preview(slug: str) -> str:
```

Comment:
Can we use settings.url_mh_integrations_preview() instead of this?

Reply:
No, because the `config.app_url` is not initialized when the parametrization data is initialized. I'll add a comment